srcpkgs/qmk/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'qmk'
+pkgname=qmk
+version=0.0.27
+revision=1
+archs=noarch
+create_wrksrc=yes
+build_wrksrc="./qmk-${version}"
+build_style=python3-module
+hostmakedepends="unzip python3-setuptools"
+depends="python3-appdirs
+	python3-argcomplete
```

Comment:
Intend newlines in vars with only a single space.

Reply:
Understood, changed.